github/filecoin-project.yml

```diff
@@ -2517,16 +2444,7 @@ repositories:
     collaborators:
       admin:
         - anorth
-        - jennijuju
-        - Kubuxu
-        - masih
-        - Stebalien
-      maintain:
```

Comment:
F3 engineering team should have maintain access: @Stebalien, @kubuxu, @masih.

Reply:
Per comments above/below, we're covered by lotus-maintainers having admin.